src/TFTDashboard.cpp

```
@@ -0,0 +1,102 @@
+        // 2x2 grid: spacious layout
+        int cy = y + h / 2;
+        sprite->setTextSize(1);
+        sprite->drawString(label, x + w / 2, cy - 28);
+        sprite->setTextSize(2);
+        sprite->drawString(tray.material, x + w / 2, cy);
+        sprite->setTextSize(1);
+        sprite->drawString(weight, x + w / 2, cy + 28);
+    }
+}
+
+void TFTDashboard::render(LGFX_Sprite* sprite, const TrayDashboardState& state) {
```

Clang (14.0.6)

[warning] method 'render' can be made static

(readability-convert-member-functions-to-static)